spec/javascripts/Stats/inlinechart/inlineChart.spec.jsx

```diff
@@ -89,26 +91,13 @@ describe('<InlineChart/>', () => {
   })
 
   it('calls updateState method with total of 1', () => {
-    const data = {
```

Comment:
I think this tests is useless, after removing `data`, you are calling it with the default data object where `total: 10`, then it's just asserting `toHaveBeenCalled` but not with `data:1`

Reply:
You're right, I didn't realize the "total" was different.. the test is wrong then, it should fail. I will fix it.